src/user/dashboard/news-feed/news-feed.js

```diff
@@ -1,11 +1,239 @@
-import React, { Component } from "react";
-import { Button } from "react-bootstrap";
+import React, { Component} from "react";
+import { Button , Dropdown} from "react-bootstrap";
 import "./news-feed.scss";
-import gsoc from "../../../images/gsoc.png";
+import profileImg from "../../../svgs/evt-creator.svg";
+import upVoteImg from "../../../svgs/up.svg";
+import downVoteImg from "../../../svgs/down.svg";
+import commentIcon from "../../../svgs/comment.svg";
+import eventImg from "../../../svgs/event-img-1.svg";
+import eventImg2 from "../../../svgs/event-img-2.svg";
+
+
+const CustomToggle = React.forwardRef(({ children, onClick }, ref) => (
+  <a
+    href=""
+    ref={ref}
+    onClick={e => {
+      e.preventDefault();
+      onClick(e);
+    }}
+  >
+    {children}
+  </a>
+));
+
+const CustomMenu = React.forwardRef(
+  ({ children, style, className, 'aria-labelledby': labeledBy }, ref) => {
+    return (
+      <div
+        ref={ref}
+        style={style}
+        className={className}
+        aria-labelledby={labeledBy}
+      >
+        <ul className="list-unstyled">
+          {children}
+        </ul>
+      </div>
+    );
+  },
+);
+
+const feed = [
+  {
```

Comment:
can you load feed array from a separate file  named `json_data`

Reply:
Sure, I'll work on this. 
Thanks for the feedback.

Reply:
Also , if you could let know any other changes that may be required.

Reply:
This is good. Squash your commits as well, when you are done

Reply:
Thanks @elit-altum. How do I squash my commits ? Also if you could let me know why is it required ?